Makefile

```diff
@@ -0,0 +1,3 @@
+shfmt:
+	go get github.com/contiv-experimental/sh/cmd/shfmt
+	find . -type f -name "*.sh" -exec shfmt -w -i 4 -p {} \;
```

Comment:
We're not using spaces. We use the default with tabs from what I know.

Reply:
we should use spaces for bash scripts, thats what most editor does

Reply:
We've been using tabs for a while now for shell scripts. That's the default for shfmt. I'd prefer we keep the default.

Reply:
That's for other project, I rarely see bash script using tab after all.

Reply:
https://google.github.io/styleguide/shell.xml?showone=Indentation#Indentation